packages/datatrak-web-server/src/routes/SubmitSurveyReponse/SubmitSurveyResponseRoute.ts

```diff
@@ -4,16 +4,42 @@
  */
 import { Request } from 'express';
 import { Route } from '@tupaia/server-boilerplate';
-import { DatatrakWebSubmitSurveyResponseRequest as RequestT } from '@tupaia/types';
+import { QUERY_CONJUNCTIONS } from '@tupaia/database';
+import {
+  DatatrakWebSubmitSurveyResponseRequest as RequestT,
+  MeditrakSurveyResponseRequest,
+} from '@tupaia/types';
 import { processSurveyResponse } from './processSurveyResponse';
 import { addRecentEntities } from '../../utils';
+import { DatatrakWebServerModelRegistry } from '../../types';
 
 export type SubmitSurveyResponseRequest = Request<
   RequestT.Params,
   RequestT.ResBody,
   RequestT.ReqBody,
   RequestT.ReqQuery
 >;
+const findTasksToComplete = async (
```

Comment:
This is almost a copy of the query that is in the TaskCompletionHandler but it is for searching based on 1 survey response instead of many. I had a try at bringing them together but it was becoming more complicated than I liked. I'm open to suggestions about a nice way of doing that.